src/components/TodosList/TodosList.tsx

```diff
@@ -0,0 +1,14 @@
+import { Todo } from '../../types/Todo';
+import { TodosItem } from '../TodosItem/TodosItem';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodosList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="todoapp__main">
+      {todos.map(todo => <TodosItem todo={todo} key={todo.id} />)}
+    </section>
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings